android/java/org/chromium/chrome/browser/crypto_wallet/activities/NetworkSelectorActivity.java

```diff
@@ -79,6 +82,8 @@ public void onNetworkItemSelected(NetworkSelectorItem networkSelectorItem) {
                                 this, networkSelectorItem.getNetworkName(), customNetworks),
                         CoinType.ETH, (success) -> {
                             if (!success) {
+                                Toast.makeText(this, getResources().getString(R.string.brave_wallet_network_selection_error) +" '"+networkSelectorItem.getNetworkShortName()+"'", Toast.LENGTH_SHORT).show();
```

Comment:
nit: always prefer use string place holder with param `%1$s` and formatting (single quote), instead of concatenation using `+`.